lib/ClimaLandSimulations/src/fluxnet/site_parameters.jl

```diff
@@ -0,0 +1,87 @@
+export site_parameters
+
+"""
+    site_parameters(;
+        hetero_resp,
+        auto_resp,
+        soil,
+        soil_heat,
+        radiative_transfer,
+        energy_balance,
+        conductance,
+        photosynthesis,
+        plant_hydraulics
+        )
+
+Setup parameters for a site
+"""
+function site_parameters(;
+    hetero_resp,
+    auto_resp,
+    soil,
+    soil_heat,
+    radiative_transfer,
+    energy_balance,
+    conductance,
+    photosynthesis,
+    plant_hydraulics,
+)
+
+    κ_solid = Soil.κ_solid(
+        soil_heat.ν_ss_om,
+        soil_heat.ν_ss_quartz,
+        soil_heat.κ_om,
+        soil_heat.κ_quartz,
+        soil_heat.κ_minerals,
+    )
+    κ_dry_val = Soil.κ_dry(soil_heat.ρp, soil.soil_ν, κ_solid, soil_heat.κ_air)
+    κ_sat_frozen_val = Soil.κ_sat_frozen(κ_solid, soil.soil_ν, soil_heat.κ_ice)
+    κ_sat_unfrozen_val =
+        Soil.κ_sat_unfrozen(κ_solid, soil.soil_ν, soil_heat.κ_liq)
+    ρc_ds = FT((1 - soil.soil_ν) * 4e6) # J/m^3/K
+    conductivity_model = PlantHydraulics.Weibull{FT}(
+        plant_hydraulics.K_sat_plant,
+        plant_hydraulics.ψ63,
+        plant_hydraulics.Weibull_param,
+    )
+    retention_model =
+        PlantHydraulics.LinearRetentionCurve{FT}(plant_hydraulics.a)
+
+    return (
+        hetero_resp = hetero_resp,
+        auto_resp = auto_resp,
+        soil = soil,
```

Comment:
should soil and soil heat be different? similarly, many of the params below (k solid, k_dry, etc) are for soil